### DISCUSSION TITLE
Error mpr_sm: pOM used uninitialized.

### DISCUSSION THREAD

**Discussion by ViniciusBMoreira** [mhm-ufz/mHM#85] [Q&A] [answered]
Hello everyone,

I encountered the error "Error mpr_sm: pOM used uninitialized." while trying to run the mHM. I'm not using an initial condition for sm and this data does not participate in the calibration. What should I check?

![image](https://github.com/mhm-ufz/mHM/assets/61054098/619ca8da-2334-44fb-9db8-f762b526f822)

**Comment by rohinikumar82** [comment, marked as answer]
This is a rather strange error. Can you check you soil database and also landcover files - if they all have valid values?

**Reply by ViniciusBMoreira** [reply]
I will revisit these model inputs. Thanks!
